packages/api-client/src/views/Settings/SettingsGeneral.vue

```diff
@@ -195,6 +202,44 @@ const changeTheme = (themeId: ThemeId) => {
                 </div>
               </ScalarButton>
             </div>
+            <div class="mt-4 mb-2">
+              <p class="text-c-1 font-bold leading-[21px">Integration Themes</p>
```

Comment:
looks like a busted class not doing anything
```suggestion
              <p class="text-c-1 font-bold">Integration Themes</p>
```